### PR TITLE
Rename OSS to open source

### PR DESCRIPTION
**Pull request by tzheleznyak**
### What does this PR do?
Rename Conjur OSS mentions to Conjur Open Source

#### Change log
- [ ] The CHANGELOG has been updated, or
- [x] This PR does not include user-facing changes and doesn't require a CHANGELOG update

#### Test coverage
- [ ] This PR includes new unit and integration tests to go with the code changes, or
- [x] The changes in this PR do not require tests

#### Documentation
- [x] Docs (e.g. `README`s) were updated in this PR, and/or there is a follow-on issue to update docs, or
- [] This PR does not require updating any documentation

### CODE REVIEW

**Review by orenbm — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -48,7 +48,7 @@ The client's process logs its flow to `stdout` and `stderr`.
 + Exponential backoff is exercised when an error occurs
 + Client will re-login when certificate has expired
 
-1. Client goes through login by presenting certificate signing request (CSR) -> Server (authn-k8s running inside the appliance) injects signed client certificate out of band into requesting pod
+1. Client goes through login by presenting certificate signing request (CSR) -> Server (authn-k8s running inside the Conjur Enterprise) injects signed client certificate out of band into requesting pod
```
Comment (hilagross) [inline README.md:51]: Conjur server

**Comment by shulifink** [comment]
Here:
image

1.  Isn't it the CyberArk (not Conjur) Kubernetes Authenticator Client? (note Authenticator and Client are both capitalised.
2. BusyBox is a name of a company: it needs to be spelled the way the company spells it: BusyBox (see https://www.busybox.net/)
3.

**Comment by shulifink** [comment]
Here:
image

The Kubernetes Authenticator Client

File: README.md [hunk 2]
```diff
@@ -16,10 +16,10 @@ The Conjur Kubernetes authenticator client is designed to have a light footprint
 
 The client is configured entirely through environment variables. These are listed below.
 
-### Using conjur-authn-k8s-client with Conjur OSS 
+### Using conjur-authn-k8s-client with Conjur Open Source 
 
-Are you using this project with [Conjur OSS](https://github.com/cyberark/conjur)? Then we 
-**strongly** recommend choosing the version of this project to use from the latest [Conjur OSS 
+Are you using this project with [Conjur Open Source](https://github.com/cyberark/conjur)? Then we 
+**strongly** recommend choosing the version of this project to use from the latest [Conjur Open Source
```
Comment (shulifink) [inline README.md:22]: this sentence is a bit cumbersome. What is this trying to say?

File: README.md [hunk 3]
```diff
@@ -48,7 +48,7 @@ The client's process logs its flow to `stdout` and `stderr`.
 + Exponential backoff is exercised when an error occurs
 + Client will re-login when certificate has expired
 
-1. Client goes through login by presenting certificate signing request (CSR) -> Server (authn-k8s running inside the appliance) injects signed client certificate out of band into requesting pod
+1. Client goes through login by presenting certificate signing request (CSR) -> Server (authn-k8s running inside the Conjur Enterprise) injects signed client certificate out of band into requesting pod
 1. Client picks up signed client certificate, deletes it from disk and uses to authenticator via mutual TLS -> Server responds with auth token (retrieved via authn-local) encrypted with the public key of the client.
 1. Client decrypts the auth token and writes it to to the shared memory volume (`/run/conjur/access-token`)
```
Comment (shulifink) [inline README.md:53]: "...and writes it to to the..." - extra "to" - delete one

**Review by shulifink — COMMENTED** [review]
small comments

**Pull request merged**